en/docs/administer/logging-and-monitoring/monitoring/monitoring-tcp-based-messages.md

```diff
@@ -4,27 +4,26 @@ Users can view and monitor the messages passed along a TCP-based conversation us
 
 TCPMon is not dependent on any third party libraries. Its user interface is based on a swing UI and works on almost all platforms that support Java.
 
--   [Starting TCPMon](_Starting_TCPMon_)
--   [Message Monitoring with TCPMon](_Message_Monitoring_with_TCPMon_)
--   [Other Usages of TCPMon](_Other_Usages_of_TCPMon_)
+-   Starting TCPMon
+-   Message Monitoring with TCPMon
+-   Other Usages of TCPMon
```

Comment:
```suggestion
```

Reply:
We dont need this as the TOC is there on the right side.

Reply:
Yes. +1. Let's remove.